opentracing_shim/README.md

```diff
@@ -1,6 +1,20 @@
+[![Javadocs][javadoc-image]][javadoc-url]
 # OpenTelemetry - OpenTracing Shim
+A bridge layer from OpenTelemetry to the OpenTracing API.
+It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.
+
+## Usage
+
+1. Configure OpenTelemetry tracer
+1. Create OpenTracing tracer from OpenTelemetry tracer
```

Comment:
Probably we should add an extra section on how to manually specify a `Tracer` with the rest of the components.

Reply:
done